hoot-services/src/main/java/hoot/services/controllers/osm/map/MapResource.java

```diff
@@ -544,10 +546,10 @@ public Response getMBR(@Context HttpServletRequest request, @PathParam("mapId")
                 || (extents.get("maxlon") == null)) {
 
             // check for bbox tag in maps
-            String bboxTag = DbUtils.getMapBbox(Long.parseLong(mapId));
-            if(bboxTag != null) {
+            String boundsTag = DbUtils.getMapBounds(Long.parseLong(mapId));
+            if(boundsTag != null) {
                 //This is how UI sends a bbox `${minx},${miny},${maxx},${maxy}`
-                String[] bboxCoords = bboxTag.split(",");
+                String[] bboxCoords = boundsTag.split(",");
```

Comment:
I think this will need to handle if the `boundsTag` is a poly and return the poly extent maybe?

Reply:
Maybe it's ok, that call to the `/mbr` method only gets done if the layer bounds tag is not defined.  It's just weird that in the service code if the calculated nodes MBR comes back null the code looks for a bounds tag again.